api/database/seeders/DatabaseSeeder.php

```diff
@@ -37,7 +37,7 @@ public function run(): void
             TeamRandomSeeder::class,
             PoolRandomSeeder::class,
             UserRandomSeeder::class,
-            AssessmentResultRandomSeeder::class,
+            // AssessmentResultRandomSeeder::class,
             SearchRequestRandomSeeder::class,
```

Comment:
Still mean to leave this commented out?

Reply:
Yes. I was worried it was overriding the assessments that are deliberately set in the other seeder.